doc/posts/2023/2023_07_12_Shilpi_week7.rst

```diff
@@ -0,0 +1,29 @@
+Modifying Test Signal Generation 
+================================
+
+.. post:: July 12 2023
+    :author: Shilpi Prasad
+    :tags: google
+    :category: gsoc
+
+
+What I did this week
+~~~~~~~~~~~~~~~~~~~~
+
+One of the tasks I did this week was modify the cti_design_matrix again, as asked by my mentor to make the code more readable. The initial code was following pep8 standard but it wasn't very easy to read, but now it is.
+Also, I realized that the main reason my signals weren't matching the ground truth values before at all was because the eigen values and eigenvectors of the diffusion tensor distribution were wrong. This was because, before I tried getting D_flat by doing: np.squeeze(from_3x3_to_6x1(D)) which returned a tensor of shape ( 6, ). But in this case, it returned the diffusion tensor elements in the order : Dxx, Dyy, Dzz  and so on which isn't the correct format of input expected for "from_lower_triangular" function. So, initially, we were doing : evals, evecs = decompose_tensor(from_lower_triangular(D_flat)) where the from_lower_triangular function is returning a tensor of shape: (3,3). But then I realized that rather than calculating D_flat, we can simply do: evals, evecs = decompose_tensor(D_flat). Following this approach gave the correct value of evals and evecs. So, now we have the correct values of evals and evecs which made the signals come closer to the ground truth signals, but we still don't have the signals completely matching the ground truth signals.
```

Comment:
- eigen values :  one word -> eigenvalues
- np.squeeze(from_3x3_to_6x1(D)) :  code so backtick 
- for "from_lower_triangular". :  for the "from_lower_triangular" instead
- evals, evecs = decompose_tensor(from_lower_triangular(D_flat)) :  code so backtick 
- evals, evecs = decompose_tensor(D_flat): code so backtick
- evals, evecs: add quote